ui/lib/apps/ClusterInfo/components/HostTable.tsx

```diff
@@ -101,6 +101,27 @@ Logical Cores:  ${c.logical_cores}`
           )
         },
       },
+      {
+        name: t('cluster_info.list.host_table.columns.cpu_arch'),
+        key: 'cpu',
+        minWidth: 100,
+        maxWidth: 150,
+        onRender: (row: IExpandedHostItem) => {
+          const { cpu_info: c } = row
+          if (!c) {
+            return
+          }
+          if (c.arch === '') {
```

Comment:
How about:

```js
if (!c || !c.arch) {
```

Reply:
I rarely write frontend code, thank you for the teaching😘